### ISSUE TITLE
Nelson's Bank Card: Pin Reset

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/hq#271]
Nelson to request a pin reset for Santander debit card

**Comment by iteles** [comment]
@nelsonic Call 0800 068 7010 between 8h00 and 19h00 on weekdays

**Comment by markwilliamfirth** [comment]
Closing as Nelson confirmed via email he's been using his bank card, so it would seem the pin situation was resolved

**Comment by nelsonic** [comment]
Nope. Not "_confirmed via email_". 
What I wrote in the email is that my card was _rejected_. i.e. it's "_not working_".

I don't _understand_ is _why_ the pin required resetting in the _first_ place...?
The pin was working `before`; I used it in 2016 without any issues. and I haven't _forgotten_ it ... 
so _why_ would I need to reset it...? Why was the card "_blocked_"? 
Did someone `else` try to use my card?

Leaving issue `closed` because I have no intention of using this card again.
Much easier to spend my _own_ money on things I need than to waste any more time on this.

**Comment by markwilliamfirth** [comment]
@nelsonic this issue was from March when you said you couldn't remember you pin

If you need assistance with unblocking your card you will need to phone Santander Business support (phone number of back of card or online). They should be able to answer your question on why it was blocked. Please create a new issue in hq if you need more assistance from me on this problem or if you'd like to record the above in an issue
